### DISCUSSION TITLE
Release (for k8s OIDC auth support in vals)?

### DISCUSSION THREAD

**Discussion by rsrchboy** [helmfile/helmfile#1327] [Ideas]
Now that https://github.com/helmfile/helmfile/pull/1325 and https://github.com/helmfile/vals/pull/201 have been merged, I can pull my secrets from k8s successfully.

Would it be possible to cut a release so I can use the "official" release image/binary?  I'm not sure what the current process/schedule for this is, but it would make my life much easier :)

**Comment by yxxhero** [comment]
next two week. maybe. waitting for this feature: https://github.com/helmfile/helmfile/pull/1329

**Reply by rsrchboy** [reply]
Thanks!  I'm only bugging as it would be very useful to me.

**Reply by yxxhero** [reply]
@rsrchboy wow. do youo like this feature?
